Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -472,6 +480,45 @@ public extension UIView {
         CATransaction.commit()
     }
 
+    /// SwifterSwift: Apply Gradient Colors.
+    ///
+    ///     view.applyGradient(
+    ///         colors: [.red, .blue],
+    ///         locations: [0.0, 1.0],
+    ///         direction: .topToBottom
+    ///     )
+    ///
+    /// - Parameters:
+    ///   - colors: An array of colors defining the color of each gradient stop.
+    ///   - locations: An array of NSNumber defining the location of each
+    ///                gradient stop as a value in the range [0,1]. The values must be
+    ///                monotonically increasing.
+    ///   - direction: Enumeration type describing the direction of the gradient.
+    func applyGradient(colors: [Color]?, locations: [CGFloat]? = [0.0, 1.0], direction: GradientDirection = .topToBottom) {
```

Comment:
I don't think these should be optional, since a gradient doesn't make sense without them.
```suggestion
    func applyGradient(colors: [Color], locations: [CGFloat] = [0.0, 1.0], direction: GradientDirection = .topToBottom) {
```